ml-agents-envs/mlagents/envs/subprocess_env_manager.py

```diff
@@ -7,8 +7,14 @@
 from queue import Empty as EmptyQueueException
 from mlagents.envs.base_unity_environment import BaseUnityEnvironment
 from mlagents.envs.env_manager import EnvManager, StepInfo
-from mlagents.envs.timers import timed
-from mlagents.envs import BrainParameters, ActionInfo
+from mlagents.envs.timers import (
+    TimerNode,
+    timed,
+    hierarchical_timer,
+    reset_timers,
+    _global_timer_stack,
```

Comment:
Should we change the name of this since we're not using it as private?

Reply:
I moved access to a function instead.